include/speed_calc.h

```diff
@@ -0,0 +1,24 @@
+#ifndef __SPEED_CALC_H__
+#define __SPEED_CALC_H__
+
+#include "Arduino.h"
+#include "const.h"
+#include "IOManagement.h"
+#include "STM32TimerInterrupt_Generic.h"
+
+#define SPEED_CALC_INTERVAL_MS 50 // ms
```

Comment:
Let's change this macro to μs here, that way we won't have to do a multiply by 1000 in `speed_calc.cpp`.